modules/openapi-generator/src/main/resources/ruby-client/api_error.mustache

```diff
@@ -26,5 +26,10 @@ module {{moduleName}}
         super arg
       end
     end
+
+    # Override to_s to print variables
+    def to_s
```

Comment:
It seems good to return information on response!
However, if left as it is, the error message passed in the form below will be hidden 🤔 
```ruby
ApiError.new('some reason')
ApiError.new(message: 'some reason', code: 400)
```
I'll think about how it should be...

Reply:
In any case, if you change to_s, can you also fix the corresponding test?
(This causes CircleCI to fail)
```
Failures:

  1) Petstore::ApiClient#initialize should save the message if one is given
     Failure/Error: expect(err.message).to eq("Hello")

       expected: "Hello"
            got: "{:code=>nil, :response_headers=>nil, :response_body=>nil}"

       (compared using ==)
     # ./spec/custom/api_error_spec.rb:7:in `block (3 levels) in <top (required)>'

  2) Petstore::ApiClient#initialize should save the hash as message if no message is given
     Failure/Error: expect(err.message).to eq("{:code=>500, :response_body=>\"server error\"}")

       expected: "{:code=>500, :response_body=>\"server error\"}"
            got: "{:code=>500, :response_headers=>nil, :response_body=>\"server error\"}"

       (compared using ==)
     # ./spec/custom/api_error_spec.rb:12:in `block (3 levels) in <top (required)>'
```